invisible_cities/cities/psf_liquidata.py

```diff
@@ -0,0 +1,193 @@
+from functools import partial
+
+import numpy  as np
+import tables as tb
+import pandas as pd
+
+from invisible_cities.reco.psf_functions import create_psf
+from invisible_cities.reco.psf_functions import hdst_psf_processing
+
+from liquidata import flat
+from liquidata import get
+from liquidata import put
+from liquidata import pipe
+from liquidata import out
```

Comment:
I don't see how you provide `liquidata`.

There are two options:

+ pip in the IC conda environment
+ add its single-file implementation to the IC repo. (The implementation has been kept in a self-contained single file, for now, exactly for the purpose of making it trivial to install in this way.)

If we were using Nix, I would recommend the `pip` approach: Nix (and `direnv`) would automatically take care of ensuring that it is installed wherever it is needed, and absent where it is not needed. Without Nix, the user has to take a much more active role in managing the conda environment. Given that the IC conda environment hasn't changed for over 18 months (?), the users have probably forgotten how to do this. (I certainly have.)

Reply:
I didn't add it anywhere yet. For me, I just added the top directory to PYTHONPATH to be able to run the tests. For the actual repo we need to add it to `manage.sh`. I will do it now.